etc/pygac-fdr.yaml

```diff
@@ -3,8 +3,8 @@ controls:
     reader_kwargs:
         tle_dir: /data/avhrr_gac/tle
         tle_name: TLE_%(satname)s.txt
-        tle_thresh: 1000
-        calibration_file: /data/avhrr_gac/calibration.json
+        tle_thresh: 7
+        # calibration_file: /data/avhrr_gac/calibration.json
```

Comment:
This is just to illustrate how to change the calibration. By default pygac's default coefficients will be used.